base/features/kotlin/skeleton/maven-build/pom.xml

```diff
@@ -1,6 +1,7 @@
 <project>
   <properties>
     <kotlinVersion>1.3.50</kotlinVersion>
+    <micronautDataVersion>1.0.0.M4</micronautDataVersion>
```

Comment:
Please remove this

Reply:
this was a leftover from something i tried earlier i missed, it has been removed